### ISSUE TITLE
[Question / Feature Request] Add support for extending abstract object types

### ISSUE THREAD

**Issue by amille14** [doug-martin/nestjs-query#82]
I currently have something like the following:
```ts
@Entity()
@ObjectType()
export class User extends BaseEntity {
  @FilterableField()
  @Column()
  name: string
  //...
}
```
```ts
@ObjectType({ isAbstract: true })
export abstract class BaseEntity {
  @FilterableField(type => ID)
  @PrimaryColumn()
  id: string
  //...
}
```

The generated schema will properly include `id` and other properties from `BaseEntity` in the `User` type, however, these properties are not included in generated types related to filtering and sorting, only the properties of class `User` are available.

Is there something I'm doing wrong or is this not supported? If not, I would like to be able to separate properties of my entities into reusable classes, please consider adding support for this.

**Comment by doug-martin** [comment]
@amille14 thanks for submitting! 

This isnt currently supported, but I don't think it should be too much work to add it. I'll look into it.

**Comment by doug-martin** [comment]
This should be fixed in `v0.8.3`. Let me know if you have any issues, thanks again for reporting!
